src/command/pack.rs

```diff
@@ -9,7 +9,15 @@ pub fn pack(path: Option<String>, log: &Logger) -> result::Result<(), Error> {
     let crate_path = set_crate_path(path);
 
     info!(&log, "Packing up the npm package...");
-    npm::npm_pack(&crate_path)?;
+    match npm::npm_pack(&crate_path) {
+        Ok(r) => r,
+        Err(Error::Io { .. }) => {
```

Comment:
I think this match is too broad. I only want to catch os error 2: directory not found. I don't want to swallow other IO errors. Investigate how to do that.